<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-RP7PCHJ7" failures="2" tests="15" name="Test" time="0.095" errors="0" timestamp="2025-01-19T15:52:53 GMT-06:00">
  <testcase classname="testcases.LoginTest" name="checkUser" time="0.005"/>
  <testcase classname="testcases.ParameterizesTestcases" name="create" time="0.001"/>
  <testcase classname="testcases.ParameterizesTestcases" name="create" time="0.002"/>
  <testcase classname="testcases.ParameterizesTestcases" name="create" time="0.0"/>
  <testcase classname="testcases.ParameterizesTestcases" name="delete" time="0.001"/>
  <testcase classname="testcases.ParameterizesTestcases" name="delete" time="0.002"/>
  <testcase classname="testcases.ParameterizesTestcases" name="delete" time="0.0"/>
  <testcase classname="testcases.LoginTest" name="doLogin" time="0.002"/>
  <testcase classname="testcases.ValidateUserRegTest" name="validate" time="0.006">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [PQR] but found [ABC],
	Custom Error Message did not expect to find [true] but found [false],
	404 Error">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [PQR] but found [ABC],
Custom Error Message did not expect to find [true] but found [false],
404 Error
at testcases.ValidateUserRegTest.validate(ValidateUserRegTest.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- validate -->
  <testcase classname="testcases.SearchAndBuyPhone" name="search" time="0.001">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testcases.SearchAndBuyPhone.search(SearchAndBuyPhone.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- search -->
  <testcase classname="testcases.PassingValuesTest" name="search" time="0.001"/>
  <testcase classname="testcases.SearchAndBuyPhone" name="selectPhone" time="0.0">
    <skipped/>
  </testcase> <!-- selectPhone -->
  <testcase classname="testcases.PassingValuesTest" name="selectPhone" time="0.001"/>
  <testcase classname="testcases.SearchAndBuyPhone" name="checkout" time="0.0">
    <skipped/>
  </testcase> <!-- checkout -->
  <testcase classname="testcases.PassingValuesTest" name="checkout" time="0.001"/>
</testsuite> <!-- Test -->
